docs/Documentation/Drivers/Image/Inputs.md

```diff
@@ -13,3 +13,4 @@ And a few more are optional:
 *   [Constants File](../../Constants.md): Model parameters that are constant in time and space.
 *   [Initial State File](StateFile.md): Moisture storages (soil moisture, snow pack, etc), energy storages (soil temperatures, etc) and other information describing the current state of the system. A state file saved from a previous VIC simulation may be used as the initial state for another run.
 *   [Lake/Wetland Parameter File](LakeParam.md): File containing lake model parameters. By default, VIC does not simulate lakes or other impoundment of surface water.
+*   [Routing Parameter File](RvicParam.md): File containing the parameters necessary to perform the Lohmann routing.
```

Comment:
Add a sentence that states that by default, VIC does not perform any streamflow routing.